.github/workflows/tests_fe.yml

```diff
@@ -0,0 +1,26 @@
+name: Testing FE
+
+on: pull_request
+
+jobs:
+  build_and_test:
+    name: Testing # job name (unique id)
+    runs-on: ubuntu-latest # on which machine to run
+    steps: # list of steps
+      - name: Install NodeJS
+        uses: actions/setup-node@v2
+        with:
+          node-version: '14'
```

Comment:
Could we run tests over the newer node version?
https://nodejs.org/en/about/previous-releases

Reply:
Thank you for feedback. It was very usefull! I've changed node version to 18 both in tests and eslint.